arviz/plots/bpvplot.py

```diff
@@ -124,7 +124,7 @@ def plot_bpv(
         parameters. If flatten is defined and flatten_pp is None, then ``flatten_pp=flatten``.
     legend : bool
         Add legend to figure. By default True.
-    ax : numpy array-like of matplotlib axes or bokeh figures, optional
+    ax : 2D array-like of matplotlib _axes or bokeh _figure, optional
```

Comment:
```suggestion
    ax : 2D array-like of matplotlib_axes or bokeh_figure, optional
```

There are spaces between `matplotlib_axes` and `bokeh_figure` aliases. Same happens below in the Returns section.